### PR TITLE
try catch Object.defineProperty to fix IE8 issue

### PR DESCRIPTION
**Pull request by kaesonho**
@mridgway

### CODE REVIEW

**Comment by mridgway** [comment]
:boat:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by redonkulus** [comment]
:+1:

File: addons/index.js [hunk 1]
```diff
@@ -21,17 +21,23 @@ var movedMethods = {
...
+                        warnedOnce = true;
+                    }
+                    return movedMethods[methodName];
                 }
-                return movedMethods[methodName];
-            }
+            });
         });
-    });
+    catch (e) {
```
Comment (lingyan) [inline addons/index.js]: Hmm... Isn't this line missing a `}` at the beginning?

Comment (kaesonho) [inline addons/index.js]: oops, thx, fixed and rebased

**Comment by redonkulus** [comment]
:+1:

**Pull request merged**